src/main/java/cn/nukkit/inventory/InventorySize.java

```diff
@@ -0,0 +1,12 @@
+package cn.nukkit.inventory;
+
+public class InventorySize {
+    public static final int CHEST = 3 * 9;
+    public static final int DOUBLECHEST = 6 * 9;
+    public static final int DISPENSER = 9;
+    public static final int DROPPER = 9;
+    public static final int HOPPER = 5;
+    public static final int FURNACE = 3;
+    public static final int BREWING_STAND = 4;
+    public static final int BEACON = 3 * 9;
+}
```

Comment:
This class is not necessary, these information are already available in the `InventoryType` enum.

Example:
```java
InventoryType.DOUBLE_CHEST.getDefaultSize();
```